lib/paper_trail/has_paper_trail.rb

```diff
@@ -269,6 +270,7 @@ def touch_with_version(name = nil)
 
         attributes.each { |column| write_attribute(column, current_time) }
         save!
+        record_update(true) if self.class.paper_trail_options[:on] == []
```

Comment:
Unrelated to your discussion, I think I found a bug.  It sounds like `touch_with_version` should always create a version, but I think I found (and fixed) a situation where it doesn't: https://github.com/airblade/paper_trail/pull/563